### DISCUSSION TITLE
Correct -handling- naming of [HttpPost] actions

### DISCUSSION THREAD

**Discussion by amankkg** [Dotnet-Boxed/Templates#40] [Q&A]
Not kinda issue. Just bit confused about `GetACTION_NAME` notation, Is `Get` prefix related to http method allowed for particular action? How to handle post actions then? And why.
P.S.: I could not find any slack/gitter IM rooms related to this project.

**Comment by RehanSaeed** [comment]
Are you talking about the route name constants? They start with the HTTP method as a convention. The template does not have any POST methods but if you need them, the idea is to start with PostXxxxxx. It's just a convention, nothing more. Route names have to be unique, just having the action name is not sufficient because you can have two methods with the same action name but different HTTP methods.

I'm thinking about using slack...

**Comment by amankkg** [comment]
Yep. Thanks.
How do you deal with views? I have a few partial views. Any suggestions over simple hard-coded names?

**Comment by RehanSaeed** [comment]
I'd advise against using plain strings in your code to refer to actions/controllers/views etc. Try to use the new nameof() operator in C# 6.0 or constants where possible. The advantages are that when you rename something the rename flows through.
